### PR TITLE
PIM-5364: Inject the mass edit operation batch job code with the DI

### PR DESCRIPTION
**Pull request by grena**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N
| Added Behats                      | N
| Changelog updated                 | Y
| Review and 2 GTM                  | Y
| Micro Demo to the PO (Story only) | N
| Migration script                  | N
| Tech Doc                          | N

## About this PR
Currently, mass edit actions have a method `getBatchJobCode()` which returns the background job to execute. It has no sens to hard code it in the class & **it forces class override just to update this string** in Enterprise Edition.

This PR aims to inject this job code via the DI, as it's only configuration. This will ease integrators job & EE overrides (See EE pull requests, it cleans useless overrides).

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/config/mass_actions.yml [hunk 1]
```diff
@@ -49,7 +51,8 @@ services:
             - '@pim_catalog.localization.localizer.converter'
             - '@pim_catalog.localization.localizer.registry'
             - '@pim_enrich.filter.product_values_edit_data'
-            - %tmp_storage_dir%
+            - '%tmp_storage_dir%'
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/config/mass_actions.yml]: not sure it should have quotes

**Comment by phaseinducer** [comment]
Good idea :+1: 
GTM

**Comment by juliensnz** [comment]
Nice PR. GTM for me !

File: src/Pim/Bundle/EnrichBundle/Resources/config/mass_actions.yml (hunk 1 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/config/mass_actions.yml]: note sure too, I saw some PR that was adding quotes for vars, I can remove them, but I don't know what's the rule about it

**Comment by solivier** [comment]
GTM

**Comment by nidup** [comment]
Great PR but take care about naming, is it
 - JobInstance code, in this case $jobInstanceCode is ok
 - or Job name, in this case $jobName is ok

Job code does not exist ;)

**Pull request merged**